### ISSUE TITLE
Uglify with sourceMap includeSources

### ISSUE THREAD

**Issue by fixurface** [nozzle/react-static#1068]
if i do yarn build, with UglifyjsPlugin config:
`
new webpack.optimize.UglifyJsPlugin({
                    sourceMap: true,
                    sourceMapIncludeSources: true
                })
            );
`
I couldn't get the source map references to my original codes,

but if i do yarn build --debug, and uglify thru cli:

> for f in dist/main.*.js
do
    w=$(echo $f | cut -d'.' -f 2)
    ./libs/UglifyJS2/bin/uglifyjs ./dist/main.$w.js 
    --source-map "url='main.$w.js.map',includeSources" -o ./dist/main.$w.js
done

I'm able to get my original codes references from the source map,

So can I get sourceMap includeSources to work in a react-static build without manually uglify thru cli?

**Comment by tannerlinsley** [comment]
Ensure you're on the latest version. We were actually just solving the problem of having too many source maps (in production). #1062

**Comment by neilyoung** [comment]
@tannerlinsley Sorry for disturbing, but I would like to know, how to achieve this (attention: Newbie question)
I was creating a new blank project in order to see, if "my" PR has found the way into the npm repo and found, it was not there.

Today I repeated it, got this at the end of an EXTREMELY long yarn intstall


    cd test 

    yarn start - Start the development server
    yarn build - Build for production
    yarn serve - Test a production build locally
  
  Error: 
     ╭──────────────────────────────────────╮
     │                                      │
     │   Update available 5.9.12 → 6.3.6    │
     │   Run npm i react-static to update   │
     │                                      │
     ╰──────────────────────────────────────╯

OK I followed 

~/Documents/test $ npm i react-static

```
node-pre-gyp WARN Using needle for node-pre-gyp https download 
[fsevents] Success: "/Users/decades/Documents/test/node_modules/fsevents/lib/binding/Release/node-v57-darwin-x64/fse.node" is installed via remote
npm WARN saveError ENOENT: no such file or directory, open '/Users/decades/Documents/test/package.json'
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN enoent ENOENT: no such file or directory, open '/Users/decades/Documents/test/package.json'
npm WARN acorn-dynamic-import@4.0.0 requires a peer of acorn@^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN test No description
npm WARN test No repository field.
npm WARN test No README data
npm WARN test No license field.

+ react-static@6.3.6
added 1302 packages from 721 contributors and audited 17215 packages in 95.944s
found 0 vulnerabilities

~/Documents/test $ react-static -V
5.9.12
````

And also my new project has all the source code in release build after `yarn build` How to really get the latest update? Do I need to uninstall my global react-static installation and re-install again?
